website_menu_by_user_status/models/website_menu.py

```diff
@@ -38,3 +36,26 @@ def _compute_visible(self):
             self.is_visible = self.user_not_logged
         else:
             self.is_visible = self.user_logged
+
+
+class WebsiteMenuVisibleForSpecificGroups(models.Model):
```

Comment:
why double inheritance?

Reply:
To clearly separate the code of 2 features. Remove one class, the code still works with the other.

Reply:
you don't need it. Why should you "delete a class"?
Unless you define a reusable mixin there's no reason to do it.

Reply:
This is not the point. It's about grouping only one feature inside a class and not mixing two concepts that work in parallel.